components/builder-api/src/server/resources/pkgs.rs

```diff
@@ -45,7 +47,8 @@ use crate::{bldr_core::metrics::CounterMetric,
                                req_state,
                                Pagination,
                                Target},
-                     resources::channels::channels_for_package_ident,
+                     resources::{channels::channels_for_package_ident,
+                                 profile::fetch_license_expiration},
```

Comment:
we try not to create public functions in the resources to be accessed across other resources. Consider putting these functions in helpers.rs.